Content.Server/_DV/Mail/EntitySystems/MailSystem.cs

```diff
@@ -69,8 +71,8 @@ public sealed class MailSystem : EntitySystem
         [Dependency] private readonly StationSystem _stationSystem = default!;
         [Dependency] private readonly TagSystem _tagSystem = default!;
 
-        // DeltaV - system that keeps track of mail and cargo stats
-        [Dependency] private readonly LogisticStatsSystem _logisticsStatsSystem = default!;
+        [Dependency] private readonly LogisticStatsSystem _logisticsStatsSystem = default!; // DeltaV - system that keeps track of mail and cargo stats
```

Comment:
dont need deltav comment

Reply:
I only moved it to the end of the line, didn't add it :^)
I'll remove it rq